bigbone/src/main/kotlin/social/bigbone/api/method/MarkerMethods.kt

```diff
@@ -0,0 +1,71 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.entity.Marker
+import social.bigbone.api.entity.Markers
+import social.bigbone.api.exception.BigBoneRequestException
+
+/**
+ * Allows access to API methods with endpoints having an "api/vX/markers" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/markers/">Mastodon markers API methods</a>
+ */
+class MarkerMethods(private val client: MastodonClient) {
+    /**
+     * Get saved timeline positions.
+     * @param timeline specifies for which timelines the position markers should be returned. This value
+     *         is chosen from the [Timeline] enum. When this value is not provided, all known markers
+     *         will be returned.
+     */
+    @JvmOverloads
+    fun getMarkers(timeline: Timeline? = null): MastodonRequest<Markers> {
```

Comment:
There's a slight difference in default values between this implementation and the documentation. I agree that calling this endpoint without parameters just to get back an empty object is nonsense, so I like this behaviour of returning all markers much better.

This might need to be revisited if many more markers get added, but for the moment is fine, I guess.